pkg/reconciler/nscert/nscert.go

```diff
@@ -96,11 +97,12 @@ func (c *reconciler) reconcile(ctx context.Context, ns *corev1.Namespace) error
 	// Only create wildcard certs for the default domain
 	defaultDomain := cfg.Domain.LookupDomainForLabels(nil /* labels */)
```

Comment:
nit: line can move to L#111 (after `c.knCertificateLister.Certificates(ns.Name).List(labelSelector)`) or L#112 should move around here.